galasa-parent/galasa-boot/src/main/java/dev/galasa/boot/Launcher.java

```diff
@@ -136,6 +136,10 @@ public static void main(String[] args) {
      */
     protected void launch(String[] args) throws LauncherException, InterruptedException {
 
+        if(!validateJavaLevel()){
+            System.exit(16);
```

Comment:
Why 16 ? Just wondering...

Reply:
Also, isn't it better just to throw an exception with a decent message, rather than system exit 16, and have the user scrabble around looking for the trace ?

An exception message would surely be logged anyway, and would cause a non-zero exit code ? LauncherException could also be expanded to indicate the exception code you wanted to deliver, and do it all at the higher level ?

Calls like System.exit make it v. hard to unit test anything... as the tests would crap out also.

Reply:
```
    public static void main(String[] args) {
        Launcher launcher = new Launcher();
        try {
            launcher.launch(args);
        } catch(Exception e) {
            logger.error("Exiting launcher due to exception",e);
            System.exit(16);
        }
        System.exit(0);
    }
```

Just throwing an exception would cause System.exit(16) at a higher level anyway yes ?
Plus we'd get a stack trace in the log ?